app/policies/admin/aptc_policy.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Admin
+  # Administrative APTC changes permission checking.
+  class AptcPolicy < ApplicationPolicy
+    def update?
+      individual_market_admin_can?(:can_edit_aptc)
```

Comment:
this should be `staff_can_edit_aptc?`; we will have these methods available in application policy, and we're explicitly defining one for each permission.